buckmaxwell/zippy
# Hyphen support in parsing zipcodes

## Test instructions

* Open an `ipython` session at the repo's parent directory and ensure the below works for you.

```
In [1]: import zipcode

In [2]: myzip = zipcode.isequal('44102')

In [3]: myzip.state
Out[3]: u'OH'

In [4]: myzip = zipcode.isequal('44102-123a')

In [5]: myzip.state
Out[5]: u'OH'

In [6]: myzip.city
Out[6]: u'CLEVELAND'
```

- [ ] All works, no errors.

That's all, thanks!
